src/sidebar/helpers/annotation-metadata.js

```diff
@@ -123,11 +126,21 @@ export function isReply(annotation) {
   return (annotation.references || []).length > 0;
 }
 
-/** Return `true` if the given annotation is new, `false` otherwise.
+/**
+ * Return true if the given annotation has been saved to the backend and assigned
+ * an ID.
  *
- * "New" means this annotation has been newly created client-side and not
- * saved to the server yet.
+ * @param {Annotation} annotation
+ * @return {annotation is SavedAnnotation}
+ */
+export function isSaved(annotation) {
+  return !!annotation.id;
+}
+
+/**
+ * Return true if an annotation has not been saved to the backend.
  *
+ * @deprecated - Use {@link isSaved} instead
```

Comment:
Various `isNew(annotation)` calls should be replaced with `!isSaved(annotation)`. This function has been left to facilitate gradual migration.

Reply:
This is nicely handled. Thanks! I do notice that the amount of usage of `isNew` is relatively minimal; converting to `!isSaved` could be a pretty quick follow-up step.